### PR TITLE
chore(deps): 1.2.x bump plugin versions

### PR DESCRIPTION
**Pull request by Fortune-Ndlovu**
## Description

[List](https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins/imports/package.json) of plugins bumped in 1.2.x:
- [x] @janus-idp/backstage-plugin-3scale-backend update to 1.5.16, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-3scale-backend?activeTab=versions)
- [x] @janus-idp/backstage-plugin-aap-backend update to 1.6.16, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-aap-backend/v/1.9.3?activeTab=versions)
- [x] @janus-idp/backstage-plugin-acr update to 1.4.14, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-acr?activeTab=versions)
- [x] @janus-idp/backstage-plugin-analytics-provider-segment update to 1.4.10, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-analytics-provider-segment?activeTab=versions)
- [x] @janus-idp/backstage-plugin-jfrog-artifactory update to -1.4.11- 1.4.12  [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-jfrog-artifactory?activeTab=versions)
- [ ]  update to  "@janus-idp/backstage-plugin-keycloak-backend": "1.9.13"
- [ ]  update to  "@janus-idp/backstage-plugin-nexus-repository-manager": "1.6.11"
- [ ]   update to  "@janus-idp/backstage-plugin-ocm-backend": "4.0.10"
- [x] @janus-idp/backstage-plugin-ocm update to 4.1.9 [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-ocm?activeTab=versions)
- [x] @janus-idp/backstage-plugin-quay update to 1.7.9 [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-quay?activeTab=versions)
- [x] @janus-idp/backstage-plugin-rbac current version is 1.24.1, 1.24.2 update not available [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-rbac?activeTab=versions)
- [x] @janus-idp/backstage-plugin-tekton update to 3.7.8 [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-tekton?activeTab=versions)
- [x] @janus-idp/backstage-plugin-topology update to 1.21.11 [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-topology?activeTab=versions)
- [x] @janus-idp/backstage-plugin-argocd current version is 1.2.4, 1.2.5 update is not available [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-argocd?activeTab=versions)
- [x] @janus-idp/backstage-scaffolder-backend-module-quay update to 1.4.13, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-quay?activeTab=versions)
- [x] @janus-idp/backstage-scaffolder-backend-module-regex update to 1.4.13, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-regex?activeTab=versions)
- [x] @janus-idp/backstage-scaffolder-backend-module-servicenow update 1.4.15, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-servicenow?activeTab=versions) 
- [x] @janus-idp/backstage-scaffolder-backend-module-sonarqube update 1.4.13, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-sonarqube?activeTab=versions)
- [x] @janus-idp/backstage-plugin-catalog-backend-module-scaffolder-relation-processor current (**deprecated**) version is 1.0.4, [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-catalog-backend-module-scaffolder-relation-processor?activeTab=versions)

## Which issue(s) does this PR fix

- Fixes #? https://issues.redhat.com/browse/RHIDP-4217

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by nickboldt** [comment]
- [x] argocd:

checking janus plugins I see 
* https://github.com/janus-idp/backstage-plugins/blob/1.2.x/plugins/argocd/package.json at version 1.1.6 and 
* https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/argocd/package.json is at 1.6.2
and I see no changes in the 1.2.x branch that would warrant a re-release of the 1.1.z stream
* https://github.com/janus-idp/backstage-plugins/commits/1.2.x/plugins/argocd

If we assume that 1.2.4 IS the correct version in 1.2.x showcase, then looking at the history the commit immediately after the 1.2.4 release moved to BS 1.28 and bumped to 1.3.0, which means 1.2.4 IS the final release for the 1.2.x branch. There are no more commits after that so **no 1.2.5 is needed**:

https://github.com/janus-idp/backstage-plugins/commits/release-1.3/plugins/argocd?after=d573a77097fcf16a718969349e637297a4e71d5c+34

- [x] catalog-backend-module-scaffolder-relation-processor:

checking the history between the 1.0.1 version in the 1.2.x branch and the 1.0.3 or 1.0.4 versions... nothing has changed 

* https://github.com/janus-idp/backstage-plugins/commits/release-1.3/plugins/catalog-backend-module-scaffolder-relation-processor
* https://github.com/janus-idp/backstage-plugins/pull/1774/files#diff-ac12c0833093f7443734ae9afe5652a1f0e62c6f2fb9f49da5fb6544f230e1ed (small metadata change, alredy released as 1.0.3)

- [ ] rbac (?) :

checking janus plugins I see 
* https://github.com/janus-idp/backstage-plugins/blob/1.2.x/plugins/rbac/package.json#L3 1.20.11
* https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/rbac/package.json 1.29.6
* there might be one change that warrants a release - https://github.com/janus-idp/backstage-plugins/pull/1882 but that would be 1.20.z not 1.24.z 

If we assume that 1.24 is the correct version for 1.2.x based on the history in the 1.3 branch:
* https://github.com/janus-idp/backstage-plugins/commits/release-1.3/plugins/rbac

then this dependency bump  should warrant a 1.24.2:
* https://github.com/janus-idp/backstage-plugins/commit/8b6c249f1d2e8097cac0260785c26496a5be1a06#diff-ffda6bb6ea58fffd3458717789ee7967c6fc2f4c675d7c325a39665f9ead7ee5

**Comment by nickboldt** [comment]
```
+ @janus-idp/backstage-plugin-rbac@1.24.2
 * [new tag]           @janus-idp/backstage-plugin-rbac@1.24.2 -> @janus-idp/backstage-plugin-rbac@1.24.2
```

--> https://www.npmjs.com/package/@janus-idp/backstage-plugin-rbac/v/1.24.2

File: dynamic-plugins/imports/package.json [hunk 1]
```diff
@@ -16,23 +16,23 @@
...
+    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.4.10",
+    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.12",
     "@janus-idp/backstage-plugin-keycloak-backend": "1.9.12",
     "@janus-idp/backstage-plugin-nexus-repository-manager": "1.6.10",
-    "@janus-idp/backstage-plugin-ocm": "4.1.8",
+    "@janus-idp/backstage-plugin-ocm": "4.1.9",
     "@janus-idp/backstage-plugin-ocm-backend": "4.0.9",
-    "@janus-idp/backstage-plugin-quay": "1.7.8",
+    "@janus-idp/backstage-plugin-quay": "1.7.9",
     "@janus-idp/backstage-plugin-rbac": "1.24.1",
```
Comment (nickboldt) [inline dynamic-plugins/imports/package.json:29]:
```suggestion
    "@janus-idp/backstage-plugin-rbac": "1.24.2",
```

**Comment by nickboldt** [comment]
> @janus-idp/backstage-plugin-jfrog-artifactory update to 1.4.11 [npm registry](https://www.npmjs.com/package/@janus-idp/backstage-plugin-jfrog-artifactory?activeTab=versions)

You actually bumped it to 1.4.12.

**Review by nickboldt — APPROVED** [review]

**Comment by nickboldt** [comment]
Converting to draft until Kim's other updates are good to go, just to avoid collisions and refactorings.

File: dynamic-plugins/imports/package.json [hunk 2]
```diff
@@ -16,24 +16,24 @@
...
-    "@janus-idp/backstage-plugin-acr": "1.4.13",
-    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.4.9",
-    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.11",
+    "@janus-idp/backstage-plugin-3scale-backend": "1.5.16",
+    "@janus-idp/backstage-plugin-aap-backend": "1.6.16",
+    "@janus-idp/backstage-plugin-acr": "1.4.14",
+    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.4.10",
+    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.12",
     "@janus-idp/backstage-plugin-keycloak-backend": "1.9.12",
     "@janus-idp/backstage-plugin-nexus-repository-manager": "1.6.10",
```
Comment (kim-tsao) [inline dynamic-plugins/imports/package.json:25]:
```suggestion
    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.6.11",
```
There's a newer version: https://www.npmjs.com/package/@janus-idp/backstage-plugin-nexus-repository-manager/v/1.6.11

File: dynamic-plugins/imports/package.json [hunk 3]
```diff
@@ -16,24 +16,24 @@
...
-    "@janus-idp/backstage-plugin-ocm": "4.1.8",
+    "@janus-idp/backstage-plugin-3scale-backend": "1.5.16",
+    "@janus-idp/backstage-plugin-aap-backend": "1.6.16",
+    "@janus-idp/backstage-plugin-acr": "1.4.14",
+    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.4.10",
+    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.12",
+    "@janus-idp/backstage-plugin-keycloak-backend": "1.9.13",
+    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.6.11",
+    "@janus-idp/backstage-plugin-ocm": "4.1.9",
     "@janus-idp/backstage-plugin-ocm-backend": "4.0.9",
```
Comment (nickboldt) [inline dynamic-plugins/imports/package.json:27]:
```suggestion
    "@janus-idp/backstage-plugin-ocm-backend": "4.0.10",
```

Comment (kim-tsao) [reply dynamic-plugins/imports/package.json:27]: > 

This is the plugin with issues that's causing e2e breakage.  See https://github.com/janus-idp/backstage-showcase/pull/1735

**Comment by kim-tsao** [comment]
closing as it's not needed anymore.  superseded by https://github.com/janus-idp/backstage-showcase/pull/1934

**Pull request closed without merging**